packages/dd-trace/test/platform/node/index.spec.js

```diff
@@ -337,28 +337,28 @@ describe('Platform', () => {
       describe('prefix', () => {
         it('should support fixarray', () => {
           const length = 0xf
-          const array = new Array(length)
-          const prefixed = msgpack.prefix(array)
+          const array = Buffer.allocUnsafe(5)
```

Comment:
I'd rename the variable as this made me pause for a bit.

Reply:
Ok